lib/plugins/aws/package/compile/events/schedule.js

```diff
@@ -147,7 +171,7 @@ class AwsCompileScheduledEvents {
                   scheduleNumberInFunction
                 );
 
-              const scheduleTemplate = `
+              const eventBusScheduleTemplate = `
```

Comment:
Instead of configuring _both_ and then choosing which one should be used.

Let's from a start configure just one that should be used